src/routes/login/Login.js

```diff
@@ -1,70 +1,92 @@
-import React, { memo, useCallback, useState } from 'react';
+import React, { useCallback } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
 import useStyles from 'isomorphic-style-loader/useStyles';
+import { Field, reduxForm } from 'redux-form';
+import CustomFieldLogin from './components/CustomFieldLogin/CustomFieldLogin';
 import s from './Login.css';
-import Input from './components/Input';
-import LoginButton from './components/LoginButton';
+import LoginButton from './components/LoginButton/LoginButton';
+import { login, registration, alertCreator } from '../../actions/login';
+import Alert from './components/Alert/Alert';
+
+const validate = value => {
+  const errors = {};
+  const minLengthPassword = 5;
+  if (!value.email) {
+    errors.email = 'Required';
+  } else if (value.email.length < minLengthPassword) {
+    errors.email = `Bro, not enough characters for a valid email`;
+  } else if (!value.email.match(/^[\d.a-z-]+@[\da-z-]{2,}.[a-z]{2,}$/i)) {
+    errors.email = 'Please, check your email, it looks invalid ';
+  }
+  if (!value.password) {
+    errors.password = 'Required';
+  } else if (value.password.length < minLengthPassword) {
+    errors.password = `Password must have at least 5 characters, you typed only ${value.password.length}`;
+  }
+
+  return errors;
+};
 
 function Login() {
+  const formLoginState = useSelector(state => state.form.Login);
```

Comment:
what if your `form` is suddenly `null`? Write `useSelector(property('form.Login`))` to avoid

Reply:
Same below